src/components/Upload.js

```diff
@@ -239,30 +255,42 @@ const Upload = ({ modalOpen, setModalOpen, setUploadConfirmed }) => {
             <Form.Field
               label="Define any key terms in this dataset:"
               control={TextArea}
-              //required
+              required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, key_terms: e.target.value}})}
+              error={errors.keyTerms}
             />
             <Form.Field
               label="Who is the primary audience? "
               control={TextArea}
-              //required
+              required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, primary_audience: e.target.value}})}
+              error={errors.primaryAudience}
             />
             <Form.Field
               label="What is the purpose of this dataset and why does it exist?"
               control={TextArea}
-              //required
+              required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, purpose: e.target.value}})}
+              error={errors.purpose}
             />
             <Form.Field
               label="What kind of decisions are being made with the dataset (currently and in the future)?"
               control={TextArea}
-              //required
+              required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, decisions: e.target.value}})}
+              error={errors.decisions}
             />
             <Form.Input
-              label="Who is the resident expert?"
+              label="Who can provide context around the dataset and answer questions about it?"
+              required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, resident_expert: e.target.value}})}
+              error={errors.residentExpert}
             />
             <Form.Field
               label="Where does this dataset show up (i.e. in major reports)?"
               control={TextArea}
               //required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, appearances: e.target.value}})}
```

Comment:
No error here?

Reply:
no error because this is the only field we are not requiring